starter/js/atm.js

```diff
@@ -1,15 +1,53 @@
 //Begin with the document ready function
+$( document ).ready(function() {
+});
+
+
+// START BELOW
+
+
+
+    var checkingBalance = 0;
+    var savingsBalance = 0;
 
     //Checking account deposit function
+    var depositChecking = function() {
+      var amount = parseInt(document.querySelector('#amountChecking').value);
+      document.querySelector('#amountChecking').value = "";
+
+      checkingBalance += amount;
 
+      document.querySelector('#checkingBalance').innerHTML = "$" + checkingBalance;
+
+      if (checkingBalance === 0) {
+        document.querySelector('#checkingBalance').className = "balance red";
+      }
+      else {
+        document.querySelector('#checkingBalance').className = "balance";
+
+      }
+    }
+    //Add event listener for checking deposit button
+    document.querySelector('#depositChecking').addEventListener('click', depositChecking);
       //On click of the depositChecking button
 
         //Get value from the amountChecking input field
 
+
         //Take that value and add it to the existing value in the checkingBalance div
 
+
     //Checking account withdrawl funtion
+    var withdrawChecking = function() {
+      var amount = parseInt(document.querySelector('#amountChecking').value);
+      document.querySelector('#amountChecking').value = "";
+
+      checkingBalance -= amount;
```

Comment:
You don't want to run this if the amount is greater than the balance. So you'll want a conditional like `if (checkingBalance >= amount)`. Same with savings below.